pkg/testutils/dir.go

```diff
@@ -21,7 +21,18 @@ import (
 // TempDir creates a directory and a function to clean it up at the end of the
 // test.
 func TempDir(t testing.TB) (string, func()) {
-	dir, err := ioutil.TempDir("", fileutil.EscapeFilename(t.Name()))
+	tmpDir := ""
+	if RunningUnderBazel() {
```

Comment:
Should've mentioned it earlier, but lets unexport this symbol, it's not currently used outside this pkg and probably doesn't need to be.